package.json

```diff
@@ -28,23 +28,28 @@
         "node": ">=18"
     },
     "scripts": {
-        "build": "tsc -b && cp src/sjcl/index.d.ts lib/src/sjcl/index.d.ts",
+        "build": "tsc -b",
         "test": "tsc -b test && node --experimental-vm-modules node_modules/jest/bin/jest.js --ci",
         "lint": "eslint .",
         "format": "prettier './(src|test|examples)/**/!(*.d).ts' --write",
         "bench": "tsc -b bench && node ./lib/bench/index.js",
         "examples": "tsc -b examples && node ./lib/examples/index.js",
-        "clean": "rimraf lib coverage"
+        "clean": "rimraf lib coverage",
+        "sjcl": "cd node_modules/sjcl && concat -o sjcl.js core/sjcl.js core/aes.js core/bitArray.js core/codecString.js core/codecBase64.js core/codecBytes.js core/sha256.js core/random.js core/bn.js core/exports.js",
+        "prepack": "npm run sjcl && npm run build",
+        "prepare": "npm run sjcl && npm run build"
```

Comment:
is prepare always run when the library is imported? the issue I faced in the past was `@cloudflare/blindrsa-ts` would not work when imported indirectly

Reply:
I could use some help to achieve this.
 Either `prepare` or `postinstall` can trigger the script, however, the path to sjcl needs to be resolved.